main/conceptual/ertp/README.md

```diff
@@ -17,9 +17,9 @@ reference, it can't. For more on object capabilities, see
 ### Asset
 
 There are three kinds of assets:
-[fungible](/glossary/#fungible),
-[non-fungible](/glossary/#non-fungible), and
-[semi-fungible](/glossary/#semi-fungible).
+[fungible](../../glossary/#fungible),
+[non-fungible](../../glossary/#non-fungible), and
+[semi-fungible](../../glossary/#semi-fungible).
```

Comment:
Note to self: check that CONTRIBUTING.md notes the preference for relative links.